README.md

```diff
@@ -29,8 +29,8 @@ my_model = make_my_model()
 using LegolasFlux
 
 # We can save whatever other columns we'd like to as well as the `weights`.
-model_row = ModelRow(; weights = fetch_weights(cpu(my_model)),
-                     architecture_version=1, loss=0.5)
+model_row = LegolasFlux.ModelV1(; weights = fetch_weights(cpu(my_model)),
+                                architecture_version=1, loss=0.5)
```

Comment:
huh, did we used to export ModelRow?  Shoudl we?

Reply:
also, this will now silently drop the loss right?  sicne it's not included in the schema

Reply:
Ohhh right, it will drop the loss. We did not export it; that it wasn't qualified here was a bug